src/components/Cards.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'bootstrap/dist/js/bootstrap.bundle.min';
```

Comment:
Please import the the bootstrap assets only in the index.js file as explained in the following tutorials:

https://create-react-app.dev/docs/adding-bootstrap/
https://blog.logrocket.com/how-to-use-bootstrap-with-react-a354715d1121/